configs/components/openssl-1.1.1.rb

```diff
@@ -64,6 +64,10 @@
       sslflags = '386'
     elsif platform.architecture =~ /64$/
       target = 'linux-x86_64'
+    elsif platform.architecture =~ /aarch/
+      # OpenSSL fails to work on aarch unless we turn down the compiler optimization.
+      # See PA-2135 for details
+      cflags += " -O2"
```

Comment:
Note that the original patch failed when upgrading to OpenSSL 1.1.1. Instead of creating a new patch based off that, I figured passing the cflags this way is cleaner and it gives us what we want.